README.md

```diff
@@ -21,24 +21,30 @@ Suggestions and contributions are welcome! I'm happy to receive feature requests
 
 Dependencies:
 * [Elm `0.19`](https://guide.elm-lang.org/install.html)
-* [Node](https://nodejs.org/en/download/) (developed with Node `10.8.0` and npm `6.4.0`)
+* [`nvm` (Node version manager)](https://github.com/nvm-sh/nvm#installing-and-updating)
 
 To run locally:
-```
+```bash
 git clone git@github.com:kfrn/reel-time.git
 cd reel-time/
-npm install
+
+# Install dependencies
+nvm install $(cat .nvmrc)  # Will also install npm
```

Comment:
You can omit `$(cat .nvmrc)`. When no version is specified, nvm will automatically use the version from `.nvmrc`.

Reply:
Ah yes you are right! Updated :+1: